CGPs/0011.md

```diff
@@ -0,0 +1,35 @@
+# CGP [0011]: Increase the gasLimit to 13M
+
+- Date: 2020-09-23
+- Author(s): @nambrot
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+This proposal sets the gasLimit parameter of the Celoblockchain to 13M from 10M. This change is necessary to accomodate the contract deploy of Celo core smart contracts whose size has increased since the initial deploy.
```

Comment:
The size actually hasn't meaningfully increased, the BGL was 20M at the time of contract deployment